lib/src/main/java/org/team100/lib/motor/drive/Falcon6DriveMotor.java

```diff
@@ -282,7 +280,7 @@ public void resetPosition() {
     public void periodic() {
         // m_rawPosition = m_motor.getSelectedSensorPosition();
         m_rawPosition = m_motor.getPosition().getValueAsDouble();
-        m_velocityRev_S = m_motor.getVelocity().getValueAsDouble();
+        m_velocityRev_S = m_motor.getVelocity().getValueAsDouble()/m_gearRatio;
```

Comment:
it seems like you're redefining the meaning of the m_velocityRev_S field.  it used to be velocity of the motor; now you're saying it's velocity of the output shaft, but this meaning is inconsistent with everything else in this class.